tf_encrypted/keras/layers/dense.py

```diff
@@ -0,0 +1,121 @@
+# pylint: disable=arguments-differ
+"""Dense (i.e. fully connected) Layer implementation."""
+from typing import Union, Optional
+import numpy as np
+import tensorflow as tf
+from tensorflow.python.keras import initializers
+
+from tf_encrypted.keras.engine.base_layer import Layer
+from tf_encrypted.keras import activations
+from tf_encrypted.protocol.pond import PondPublicTensor, PondPrivateTensor
```

Comment:
this is bad. seems the only purpose is typing, in which case we should make sure that typing is really needed